Sources/Controllers/Cells/OAPublicTransportShieldCell.h

```diff
@@ -23,7 +23,4 @@
 -(void) setData:(SHARED_PTR<TransportRouteResult>)data;
 -(void) needsSafeAreaInsets:(BOOL)needsInsets;
 
-+ (CGFloat) getCellHeight:(CGFloat)width route:(SHARED_PTR<TransportRouteResult>)route;
```

Comment:
Transport shields are broken, revert and fix correctly!

Reply:
Edit: perhaps simply setting the background color and removing the separator will fix it, but still you need to check such things!